app/src/main/res/values-fr/strings.xml

```diff
@@ -118,27 +118,27 @@
     <string name="custom_input_style_note_message" msgid="8826731320846363423">"Vous devez activer votre style de saisie personnalisé avant de l\'utiliser. Voulez-vous le faire maintenant ?"</string>
     <string name="enable" msgid="5031294444630523247">"Activer"</string>
     <string name="not_now" msgid="6172462888202790482">"Pas maintenant"</string>
-    <string name="custom_input_style_already_exists" msgid="8008728952215449707">Le style de saisie suivant existe déjà: <xliff:g id="INPUT_STYLE_NAME">%s</xliff:g></string>
```

Comment:
@BoFFire thanks for your contribution, FYI I reverted your change from "style" to "méthode", as "méthode" is used to describe an actual keyboard application, where "style" refers to a keyboard configuration (layout etc...).

Reply:
Ok thanks 🙏